CONTRIBUTING.md

```diff
@@ -17,24 +17,8 @@ Let us know if these look good to you.
 You can use `npm test` to validate data against the schema. You might need to install the devDependencies using `npm install`.
 The JSON data is validated against the schema using [`ajv`](http://epoberezkin.github.io/ajv/).
 
-### Optional: Validate/cross-reference against Web API Confluence Dashboard
-If the feature you're interested in is a JavaScript API, you can cross-reference data against the [Web API Confluence Dashboard](https://web-confluence.appspot.com/) using the `confluence` npm script. This script will overwrite data in your current working tree according to data from the dashboard.
-
-**Note: Web API Confluence Dashboard data should not be regarded as perfect knowledge of JavaScript APIs.**
-The dashboard derives its data from the JavaScript object graph on a sample page loaded in each browser. For example, an own property named `URL` on `Document.prototype` implies the "`Document` interface has a member named `URL`. For various reasons, not all APIs are exposed on JavaScript prototypes, even when the API is available in the browser.
-
-Examples:
-
-```shell
-# Load confluence data for ServiceWorker
-npm run confluence -- --interfaces=ServiceWorker
-
-# Fill in missing/ambiguous Firefox data on known interfaces
-npm run confluence -- --browsers=firefox --fill-only
-
-# Print documentation on full list of options
-npm run confluence -- --help
-```
+## Generating data using the Web API Confluence Dashboard
+If the feature you're interested in is a JavaScript API, you can cross-reference data against the [Web API Confluence Dashboard](https://web-confluence.appspot.com/) using the `confluence` command. This command overwrites data in your current working tree according to data from the dashboard. See [Using Confluence](USING-CONFLUENCE.md) for instructions.
```

Comment:
* The linked page doesn't actually use the word "Dashboard" so maybe "the Web API Confluence Dashboard" → "Web API Confluence"
* `confluence` → `npm run confluence`
* "See Using Confluence" → "Read Using Confluence" (inclusive language)

```suggestion
If the feature you're interested in is a JavaScript API, you can cross-reference data against [Web API Confluence](https://web-confluence.appspot.com/) using the `npm run confluence` command. This command overwrites data in your current working tree according to data from the dashboard. Read [Using Confluence](USING-CONFLUENCE.md) for instructions.
```

Reply:
* Dashboard fixed.
* This is an intro, not the tutorial.
* 'Read' isn't inclusive either. The blind listen to pages with screen readers.